src/commons/TestUtils.ts

```diff
@@ -27,4 +30,24 @@ export namespace TestUtils {
   export function extractNumbersFromString(str: string): string {
     return str.replace(/[^0-9]/g, '');
   }
+
+  /**
+   * Return generic type of data by provided file path
+   * path File located in wdio config file
+   * example of using provided in spec src/test/specs/GetTestDataFileSpec.ts
+   * @param dataTag type of file
+   *  configDataFilePath will take from wdio.config file
+   */
+  export function getData<T>(dataTag: string = process.env.TEST_DATA_TAG): T {
+    // @ts-ignore
+    const dataFilename = browser.config.configDataFilePath;
+    const dataFilePath = path.resolve(process.cwd(), dataFilename);
+    if (dataFilename === undefined) {
+      throw new Error('Path to data file is incorrect');
+    }
+    console.log(`Getting data from file ${dataFilePath}`);
+    const data: T = require(dataFilePath);
+    console.log(`Received a data from file by provided tag ${JSON.stringify(data[dataTag])}`);
```

Comment:
Do we want to log the whole data object or only the tag name?

Reply:
add